.github/ISSUE_TEMPLATE/bug.md

```diff
@@ -0,0 +1,33 @@
+---
+name: 'Bug Report'
+about: 'Report a bug, something does not work as it supposed to'
+title: '[bug] SHORT DESCRIPTION'
+labels: bug
+---
+
+<!--
+  Please don't forget to update the issue title.
+  Include all applicable information to help us reproduce your problem.
+
+  To help us debug your issue please explain:
+-->
+
+### Environment Details (include every applicable attribute)
+  * Operating System+version: **Linux Ubuntu 18.04**
+  * Compiler+version: **GCC 8**
+  * Docker image (if applicable): **conanio/gcc8**
+  * Conan version: **conan 1.18.0**
+  * Python version: **Python 3.7.4**
+
+### Steps to reproduce (Include if Applicable)
+
+
+
+### Logs (Include/Attach if Applicable)
```

Comment:
I don't know if the logs section is applicable to most of the cases... 
Maybe add a little note in the header asking for the logs in case they are relevant?

Reply:
Logs are relevant IMO, but I agree a better specification would be nice

Reply:
done